test/test_dockerpy/README.md

```diff
@@ -2,29 +2,43 @@
 
 Python test suite to validate Podman endpoints using dockerpy library
 
+Validate :-
+
+Make sure podman binary exist under libpod/bin
+
+Validate Python 3.6+ installed using `python --version `
+
+Validate docker-py module exist by using `python list`. If missing install.
+
+
 Running tests
 =============
 To run the tests locally in your sandbox:
 
-#### Run the entire test
+cd into the libpod folder and run one of the following commands:-
+
+Note:- The test can be ran as root or rootless i.e sudo is optional.
```

Comment:
If sudo is optional, I recommend not including it in the sample commands below. Always start with the bare minimum privilege; assume the user can figure out sudo on their own if desired.